ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/questionMark.html

```diff
@@ -1,4 +1,4 @@
 <template data-sly-template.questionMark="${@ componentId, longDescription, bemBlock}">
-  <button class="${bemBlock}__questionmark" data-sly-test="${longDescription}">
+  <button id = "${componentId}-questionmark" class="${bemBlock}__questionmark" data-sly-test="${longDescription}">
```

Comment:
we should avoid id.

Reply:
without the id it is not possible to get this element.